tests/e2e/network/service_annotations.go

```diff
@@ -436,7 +436,10 @@ var _ = Describe("Service with annotation", func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 
-			// TODO: clean up PIPPrefix
```

Comment:
Could you please split this `defer func()`? Move `utils.DeletePIPPrefixWithRetry` right after `utils.WaitCreatePIPPrefix`. Move `DeleteServcie` after Service creation L451.

Reply:
After L487 Service update, please remeber to clean up the first PIP. If not, Service deletion won't consider the first PIP.

Reply:
Could you please use `wait.Poll` for L462, L496 loops?

Reply:
fixed. PTAL